frontend/components/Editor.js

```diff
@@ -1315,9 +1333,6 @@ patch: ${JSON.stringify(
                         <${Notebook}
                             notebook=${this.state.notebook}
                             cell_inputs_local=${this.state.cell_inputs_local}
-                            on_update_doc_query=${this.actions.set_doc_query}
-                            on_cell_input=${this.actions.set_local_cell}
-                            on_focus_neighbor=${this.actions.focus_on_neighbor}
```

Comment:
Nice find! We are we passing this down 🤦?

Reply:
We were yes - no longer!